src/AuthService/Application/Core/Application/Commands/SignInCommand.cs

```diff
@@ -0,0 +1,108 @@
+﻿using System.IdentityModel.Tokens.Jwt;
+using System.Security.Claims;
+using System.Text;
+using Application.Abstractions;
+using Application.DTO;
+using Domain;
+using Domain.Abstractions;
+using Domain.Entities;
+using Domain.Options;
+using FluentValidation;
+using MediatR;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Options;
+using Microsoft.IdentityModel.Tokens;
+
+namespace Application.Commands;
+
+public sealed record SignInCommand(string Email, string Password) : IRequest<TokenDto>;
+
+public sealed class SignInCommandHandler(
+    UserManager<User> userManager,
+    IRefreshTokenRepository refreshTokenRepository,
```

Comment:
looks strange to have both refreshTokenRepository and refreshTokenService at one place. If some of it is from the library I would suggest to place it under the custom repo/interface and just have one place to interact with refresh tokens